### ISSUE TITLE
Make it possible to freeze a page in place without requiring the URL

### ISSUE THREAD

**Issue by biancadanforth** [erikrose/fathom-fox#10]
**TL;DR:**
Some pages' views can change without changing the URL. It'd be nice if it were possible to freeze a page in its current view, regardless of whether or not that view is reflected in the URL for the page.

---

I was looking through some Price Wise issues and came across [this comment](https://github.com/mozilla/price-wise/issues/268#issuecomment-438841394) observing that Fathom is extracting a "product" on a variation of [this page](https://www.walmart.com/ip/Microsoft-Surface-Book13-5-8GB-128GB-Intel-Core-i5-processor-Windows-10-Pro/52497343) (screenshot below) when there should be none.

![screen shot 2018-11-30 at 1 47 30 pm](https://user-images.githubusercontent.com/17437436/49316688-8cdf6800-f4a6-11e8-8c5d-cf5d5a3d56bc.png)

To reproduce the exact issue, however, I need to select a combination of options from that page for which there is no product available, for example 512GB Hard Drive and 8GB RAM. Then I see this:

![screen shot 2018-11-30 at 1 49 26 pm](https://user-images.githubusercontent.com/17437436/49316751-cf08a980-f4a6-11e8-8024-19a85fe33a9f.png)

Unfortunately, making these changes to my product selection does not change the URL for the page. This is quite common on product pages where the user must refine their choice by selecting from a variety of options. With the current version (2.3) of FathomFox, I cannot freeze this updated version of the page where the extraction bug exists, since the URL alone will only load the original view (first screenshot), not the updated view (second screenshot) with my selections made.

In order to build a corpus of training and test pages where Fathom correctly extracts (and in this case, correctly does NOT extract) information on pages like this, whose views can change without altering the URL, it would be helpful if it were possible to freeze a page "in-place".

Granted, in Price Wise's particular case, there are other challenges such as how to know when to trigger re-extraction of the page after the user has made their selections and how to perform background updates which are also currently based on the page's URL.

Either way, this seems like a broadly useful capability as more and more webpages become web apps.

**Comment by erikrose** [comment]
You can! Just open the developer tools, go to the Fathom panel, and hit Save. :-D
